### PR TITLE
Added 5 organic reactions

### PR DESCRIPTION
**Pull request by kavienanj**
Hacktoberfest contribution!

### CODE REVIEW

File: reactions.json [hunk 1]
```diff
@@ -208,6 +208,34 @@
     "KI": "iodobenzene",
     "HBF4 / (Heat)": "fluorobenzene"
   },
+  "but-2-ene": {
+    "H2O / dil H2SO4": "butan-2-ol",
+    "PCC": "butanone"
```
Comment (CodeWithSwastik) [inline reactions.json]: Are you sure this is correct?

File: reactions.json [hunk 2]
```diff
@@ -208,6 +208,34 @@
     "KI": "iodobenzene",
     "HBF4 / (Heat)": "fluorobenzene"
   },
+  "but-2-ene": {
+    "H2O / dil H2SO4": "butan-2-ol",
+    "PCC": "butanone"
+  },
+  "acetone": {
```
Comment (CodeWithSwastik) [inline reactions.json]: Please rename this to propanone

File: reactions.json [hunk 3]
```diff
@@ -208,6 +208,34 @@
...
+  "ethanal": {
+    "NaOH": "3-hydroxybutanal",
+    "I2 (343K)": "but-2-enal",
+    "Br2 / CCl4": "2,3-dibromobutanal",
+    "alc. KOH": "but-2-ynal"
+  },
+  "benzoic acid": {
+    "LiAlH4": "phenylmethanol",
+    "PCl5": "benzyl chloride",
+    "Mg / Dry ether": "benzylmagnesium chloride",
```
Comment (CodeWithSwastik) [inline reactions.json]: How did it form a chloride here? neither the substrate nor the reagent has chlorine.

**Review by CodeWithSwastik — CHANGES_REQUESTED** [review]

File: reactions.json (hunk 1 above)
Comment (kavienanj) [reply reactions.json]: Sorry for the mistake, the product should be butan-2-one

File: reactions.json (hunk 2 above)
Comment (kavienanj) [reply reactions.json]: Sure, its actually propan-2-one

File: reactions.json (hunk 3 above)
Comment (kavienanj) [reply reactions.json]: LiAlH4 first refuses the carboxylic group to alcohol, then chloride forms due to the PCl5 reagent

Comment (CodeWithSwastik) [reply reactions.json]: I think you made a mistake in understanding the json structure. Each key and value denotes a seperate reaction and its not a chain reaction as you've put it. 
What you should've written is 
```json
  "benzoic acid": {
    "LiAlH4": "phenylmethanol"
   },
  "phenylmethanol": {
      "PCl5": "benzyl chloride"
  },
  "benzyl chloride": {
      "Mg / Dry ether": "benzylmagnesium chloride"
  },
```

You'll need to edit all the 5 reactions in this format.

File: reactions.json [hunk 4]
```diff
@@ -208,6 +208,34 @@
     "KI": "iodobenzene",
     "HBF4 / (Heat)": "fluorobenzene"
   },
+  "but-2-ene": {
+    "H2O / dil H2SO4": "butan-2-ol",
+    "PCC": "butan-2-one"
+  },
+  "propan-2-one": {
```
Comment (CodeWithSwastik) [inline reactions.json]: Can you change it to propanone, the 2 is not needed.

**Review by CodeWithSwastik — CHANGES_REQUESTED** [review]

File: reactions.json (hunk 3 above)
Comment (kavienanj) [reply reactions.json]: I am so sorry. All 5 contributions are misleading. I'll fix them all again

File: reactions.json (hunk 4 above)
Comment (kavienanj) [reply reactions.json]: Definietely

**Comment by kavienanj** [comment]
I have now made the corrections according to your request

**Comment by CodeWithSwastik** [comment]
Looks good! Thanks for the contribution.

**Pull request merged**